domain/application/state/types.go

```diff
@@ -97,6 +97,10 @@ type unitName struct {
 	Name coreunit.Name `db:"name"`
 }
 
+type unitLife struct {
```

Comment:
Is there a way to rename:
```
// applicationLife is used to get the life of an application.
type applicationLife struct {
	LifeID life.Life `db:"life_id"`
}
```
so as to not have two struts with the same contents?

Reply:
There were some discussions about this in the cross team this morning. We're quite cautious not to overload these types.

In fact, these incredibly simple structs used to query a single field should probably be moved inline